llvm/lib/Target/StubV/StubVInstrInfo.td

```diff
@@ -346,6 +346,38 @@ def SLL : Instruction {
   let mayStore = 0;
 }
 
+// WIP: I don't know how this all works
+def SRL : Instruction {
+  let Namespace = "StubV";
+  dag OutOperandList = (outs GPR:$rd); // Does this define rd, or is this a
+                               // register type that is defined elsewhere?
+  dag InOperandList = (ins GPR:$rs1, GPR:$rs2); // Likewise
+
+  let AsmString = "srl\t$rd, ${rs1}, ${rs2}";
+
+  field bits<32> Inst;
+  field bits<32> SoftFail = 0;
+  let Size = 4;
+
+  bits<5> rd;
+  bits<5> rs1;
+  bits<5> rs2;
+
+  let Inst{31-25} = 0b0000000;
+  let Inst{24-20} = rs2;
+  let Inst{19-15} = rs1;
+  let Inst{14-12} = 0b011; // I have chosen this number out of trial and error.
+                           // Where are these opcodes specified?
```

Comment:
I actually think you've got the wrong number, I had the `SRA` instruction and also took me a while to find it.

In the 20240411 spec rendering the answer is on page 574 of the PDF (says 555 on the page): https://drive.google.com/file/d/1uviu1nH-tScFfgrovvFCrj7Omv8tFtkp/view